Framework/Logger/include/Framework/Logger.h

```diff
@@ -0,0 +1,36 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+#ifndef O2_FRAMEWORK_LOGGER_H_
+#define O2_FRAMEWORK_LOGGER_H_
+
+// FIXME: until we actually have fmt widely available we simply print out the
+// format string.
+// If FairLogger is not available, we use fmt::printf directly, with no level.
+#if __has_include(<fairlogger/Logger.h>)
+#include <fairlogger/Logger.h>
+#if __has_include(<fmt/format.h>)
+#include <fmt/format.h>
+#define LOGF(level, ...) LOG(level) << fmt::format(__VA_ARGS__)
```

Comment:
Ok to me in general.

I thought a bit about the constepr point you mentioned:
We could do:
```
static inline constexpr auto myformat(const char* text) {return text;}
template <typename... Args> static inline constexpr auto myformat(const char* text, Args... args) {return text ## args...;} // or whatever syntax you need for constexpr string creation.
#define LOGF(level, ...) Log(level) << myformat(__VA_ARGS__)
```
Probably, we don't even need to make myformat constexpr, since the string will be created constexpr inside, and then the inline is sufficient.

Reply:
`text` might actually be a string, but apart from that, I get what you mean.. I will play a bit with it.

Reply:
You are right, I'd just make it a template parameter.